### ISSUE TITLE
#question / #feature: enable timeline to students

### ISSUE THREAD

**Issue by decastro-alex** [okpy/ok-client#354]
Dear friends, we're using your client for our programming classes at Imperial but we're yet to find a way to enable a student to see his/her timeline too. Is this feature available? Is it on you roadmap? If we were to enable it ourselves, any hints on where we should start hacking the source code? 

Many thanks
Alex

**Comment by colinschoen** [comment]
Hi Alex,

Are you referring to the timeline feature in `ok` server?

**Comment by Sumukh** [comment]
It's a cool idea - I think we should move this issue to the `ok` server repo - I think the issue will be mostly make sure the CSS works on the student view & making sure we aren't exposing any admin only info right now.

**Comment by decastro-alex** [comment]
@colinschoen yes, in the server. Apologies. I was a bit confused about which repo to submit a request to. 

But the request still holds: students would benefit from having that extra view. Should I repost that in the okpy/ok git repo? @colinschoen @Sumukh

**Comment by decastro-alex** [comment]
@colinschoen yes, in the server. Apologies. I was a bit confused about which repo to submit a request to. 

But the request still holds: students would benefit from having that extra view. Should I repost that in the okpy/ok git repo? @colinschoen @Sumukh 

<img width="454" alt="image" src="https://user-images.githubusercontent.com/12429626/48202164-0a0e3600-e35c-11e8-9ee4-63705c295316.png">
